netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js

```diff
@@ -345,7 +400,7 @@ const CandidatesTable = (props) => {
           sendUserFeedBack(
             currentCandidate._id,
             currentCandidate.email,
-            'Your data access request has been denied',
+            'Thank you for your interest in our Air Quality data',
```

Comment:
Updated rejection mail title to a more positive one:)

Reply:
![test email](https://user-images.githubusercontent.com/46527380/203777127-6b7894c0-6a6b-479d-a184-376f7ca024ef.png)